src/main/java/storage/Storage.java

```diff
@@ -11,9 +11,12 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class Storage {
 
+    private static Logger logger = Logger.getLogger("Storage");
```

Comment:
Let's use the same logger name as the one created in the FileLogger class so that our logs will be appended into the log file instead of printed to the console.

So the line should be:
private static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);